README.md

```diff
@@ -1,13 +1,13 @@
 # Kiwix JS
 
-## Summary
+## Summary
 Kiwix is an offline Wikipedia viewer. See the official site: http://www.kiwix.org/
 
 This is a browser extension developed in HTML5/Javascript.
 
 You can search among the article titles, and read any of them without any Internet access.
 All the content of Wikipedia is inside your device (including the images).
-It might also works with other content in the OpenZIM format: http://www.openzim.org/wiki/OpenZIM , but has been only tested on the Wikipedia ones.
+It might also works with other content in the OpenZIM format: http://www.openzim.org/wiki/OpenZIM , but has been only tested on the Wikipedia and StackExchange ones.
```

Comment:
I've done a fair amount of testing of Wikivoyage also (not just on Kiwix JS Windows, but also on Kiwix JS), so I'd feel fairly confident of adding that to the list.